### PR TITLE
Update index.md

### PR DESCRIPTION
**Pull request by Code-DJ**
Added Enable 32-Bit Applications information. See https://github.com/aspnet/AspNetCore/issues/4413

### CODE REVIEW

File: aspnetcore/host-and-deploy/iis/index.md [hunk 1]
```diff
@@ -324,6 +324,8 @@ When deploying apps to servers with [Web Deploy](/iis/publish/using-web-deploy/i
 
     ASP.NET Core runs in a separate process and manages the runtime. ASP.NET Core doesn't rely on loading the desktop CLR. Setting the **.NET CLR version** to **No Managed Code** is optional.
 
+1. For 64-bit application make sure that Enable 32-Bit Applications is set to false either under the **Set Application Pool Defaults** or under **Advanced Settings** for your specific application pool.
```
Comment (jkotalik) [inline aspnetcore/host-and-deploy/iis/index.md]: Clarify this only affects standalone deployment. If you publish for a `Framework Dependent` app, this isn't affected.

**Comment by jkotalik** [comment]
@Code-DJ I just opened a PR to address the 32-bit applications error codes. https://github.com/aspnet/Docs/pull/9846. This is a nice to have though.

**Comment by guardrex** [comment]
I'll be over in a few minutes to take a look. I'm finishing up the review on the other one now. Stand-by.

**Comment by guardrex** [comment]
@jkotalik One more :eye: on this plz.

**Pull request merged**